### PR TITLE
Add missing image, small capitalisation tweaks

### PR DESCRIPTION
**Pull request by henryhadlow**

### CODE REVIEW

File: service-manual/measurement/user-satisfaction.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ You must prompt users to give feedback at service endpoints. In the example abov
 
 You must also allow users to give feedback from anywhere in the service, in case anything goes wrong.
 
-And you must be able to show that you’re collecting user satisfaction data appropriately at your [Service Assessment](https://www.gov.uk/service-manual/digital-by-default/assessments-at-gds.html).
+And you must be able to show that you’re collecting user satisfaction data appropriately at your [Service Assessment](/service-manual/digital-by-default/assessments-at-gds.html).
```
Comment (ajlanghorn) [inline service-manual/measurement/user-satisfaction.md]: Should this sentence be conflated with the sentence above? That would get rid of the fact that the sentence starts with a conjunction.

Comment (henryhadlow) [inline service-manual/measurement/user-satisfaction.md]: I don't think there's anything wrong with starting a sentence with a conjunction. We do it pretty often on the GDS blog.

Comment (henryhadlow) [inline service-manual/measurement/user-satisfaction.md]: For example, [this post about Verify](https://gds.blog.gov.uk/2015/09/23/why-gov-uk-verify-matters/) contains 3 instances of a sentence starting with _And_.

**Comment by henryhadlow** [comment]
How does one get PRs merged these days?

**Pull request merged**